src/js/main.js

```diff
@@ -1,41 +1,47 @@
-import reqwest from 'reqwest'
 import mainHTML from './text/main.html!text'
 import share from './lib/share'
 import {pimpYouTubePlayer, getYouTubeVideoDuration} from './lib/youtube'
+import sheetToDOM from './lib/sheettodom'
 
 var shareFn = share('Interactive title', 'http://gu.com/p/URL', '#Interactive');
 
 
 export function init(el, context, config, mediator) {
-    el.innerHTML = mainHTML.replace(/%assetPath%/g, config.assetPath);
+const builder = document.createElement('div');
+builder.innerHTML = mainHTML.replace(/%assetPath%/g, config.assetPath);
 
-    [].slice.apply(el.querySelectorAll('.interactive-share')).forEach(shareEl => {
-        var network = shareEl.getAttribute('data-network');
-        shareEl.addEventListener('click', () => shareFn(network));
-    });
+    sheetToDOM('1NcSvccw77rHAncarwfeq7RGZF7yez9mP1Icf3oLMA7g', config.sheetName, builder, function callback(){
+        [].slice.apply(builder.querySelectorAll('.interactive-share')).forEach(shareEl => {
+            var network = shareEl.getAttribute('data-network');
+            shareEl.addEventListener('click', () => shareFn(network));
+        });
 
-    pimpYouTubePlayer('-Gy7poRbUHY', 'ytGuPlayer', '100%', '100%');
-    getYouTubeVideoDuration('-Gy7poRbUHY', function(duration) {
-      el.querySelector('.docs__poster--play-button').setAttribute("data-duration", duration);
-    })
+        getYouTubeVideoDuration('-Gy7poRbUHY', function(duration) {
```

Comment:
Yes, that is absolutely an option, if we are happy for editorial to populate it.